### PR TITLE
XW-923 add tests for new global navigation and adjust existing ones	

### PR DESCRIPTION
**Pull request by bognix**
**DON'T MERGE**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/ExploreWikiaDropdown.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+import org.openqa.selenium.Dimension;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class ExploreWikiaDropdown extends NewTestTemplate {
+
+  private Dimension HUBS_IN_DROPDOWN_RESOLUTION = new Dimension(768, 1024);
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/ExploreWikiaDropdown.java]: MAJOR Rename this field "HUBS_IN_DROPDOWN_RESOLUTION" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00116)

File: src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/ExploreWikiaDropdown.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  private Dimension HUBS_OUTSIDE_DROPDOWN_RESOLUTION = new Dimension(1024, 1024);
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/ExploreWikiaDropdown.java]: MAJOR Rename this field "HUBS_OUTSIDE_DROPDOWN_RESOLUTION" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00116)

File: src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/ExploreWikiaDropdown.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  private List<String> EXPECTED_LINKS_BIG_RESOLUTION =
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/ExploreWikiaDropdown.java]: MAJOR Rename this field "EXPECTED_LINKS_BIG_RESOLUTION" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00116)

File: src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/ExploreWikiaDropdown.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+          Arrays.asList("Top Communities", "Community Central", "START A WIKIA");
+  private List<String> EXPECTED_LINKS_SMALL_RESOLUTION =
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/ExploreWikiaDropdown.java]: MAJOR Rename this field "EXPECTED_LINKS_SMALL_RESOLUTION" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00116)

File: src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/GameStarLogo.java [hunk 5]
```diff
@@ -11,28 +11,16 @@
 public class GameStarLogo extends NewTestTemplate {
 
   private final static String deWikiName = "de.gta";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/GameStarLogo.java]: MAJOR Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00115)
MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/globalnav/GlobalNavigationPageObject.java [hunk 6]
```diff
@@ -124,7 +139,7 @@ public HomePageObject clickWikiaLogo() {
     if (!"prod".equals(environment) && !environment.contains("dev")) {
       WebDriverWait wait = new WebDriverWait(driver, 5);
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/globalnav/GlobalNavigationPageObject.java]: MAJOR Rename "wait" which hides the field declared at line 72. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AHiddenFieldCheck)

File: src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/GlobalNavigationWikiaLogo.java [hunk 7]
```diff
@@ -12,6 +14,9 @@
 @Test(groups = {"WikiaLogo"})
 public class GlobalNavigationWikiaLogo extends NewTestTemplate {
 
+  private final String EN_COMMUNITY = "muppet";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/GlobalNavigationWikiaLogo.java]: MAJOR Make this final field static too. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1170)
MAJOR Rename this field "EN_COMMUNITY" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00116)

File: src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/GlobalNavigationWikiaLogo.java [hunk 8]
```diff
@@ -12,6 +14,9 @@
...
+  private final String FANDOM_URL = "fandom.wikia";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/GlobalNavigationWikiaLogo.java]: MAJOR Make this final field static too. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1170)
MAJOR Rename this field "FANDOM_URL" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00116)

File: src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/HubsLinks.java [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+        import com.wikia.webdriver.common.core.Assertion;
+        import com.wikia.webdriver.common.templates.NewTestTemplate;
+        import com.wikia.webdriver.pageobjectsfactory.pageobject.HomePageObject;
+        import com.wikia.webdriver.pageobjectsfactory.pageobject.globalnav.GlobalNavigationPageObject;
+        import org.openqa.selenium.Dimension;
+        import org.testng.annotations.Test;
+
+public class HubsLinks extends NewTestTemplate {
+
+  private Dimension HUBS_IN_DROPDOWN_RESOLUTION = new Dimension(768, 1024);
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/HubsLinks.java]: MAJOR Rename this field "HUBS_IN_DROPDOWN_RESOLUTION" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00116)

File: src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/HubsLinks.java [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+  private Dimension HUBS_OUTSIDE_DROPDOWN_RESOLUTION = new Dimension(1024, 1024);
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/HubsLinks.java]: MAJOR Rename this field "HUBS_OUTSIDE_DROPDOWN_RESOLUTION" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00116)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 26 issues:
* MAJOR 18 major
* MINOR 8 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Define a constant instead of duplicating this literal "logInDropDownFB" 5 times.](https://github.com/Wikia/selenium-tests/blob/c25db3f57c6037da05a2940246e76fd0cbd93c27/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/dropdowncomponentobject/DropDownComponentObject.java#L152) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "logOut" 5 times.](https://github.com/Wikia/selenium-tests/blob/c25db3f57c6037da05a2940246e76fd0cbd93c27/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java#L712) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/c25db3f57c6037da05a2940246e76fd0cbd93c27/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java#L720) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MINOR [Define a constant instead of duplicating this literal "page loads for more than 30 seconds" 3 times.](https://github.com/Wikia/selenium-tests/blob/c25db3f57c6037da05a2940246e76fd0cbd93c27/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java#L712) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal " opened" 5 times.](https://github.com/Wikia/selenium-tests/blob/c25db3f57c6037da05a2940246e76fd0cbd93c27/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java#L213) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/c25db3f57c6037da05a2940246e76fd0cbd93c27/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java#L922) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/c25db3f57c6037da05a2940246e76fd0cbd93c27/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java#L730) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/c25db3f57c6037da05a2940246e76fd0cbd93c27/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java#L936) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/c25db3f57c6037da05a2940246e76fd0cbd93c27/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java#L711) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MINOR [Define a constant instead of duplicating this literal "%userName%" 3 times.](https://github.com/Wikia/selenium-tests/blob/c25db3f57c6037da05a2940246e76fd0cbd93c27/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java#L313) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Move the array designator from the variable to the type.](https://github.com/Wikia/selenium-tests/blob/c25db3f57c6037da05a2940246e76fd0cbd93c27/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java#L439) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1197)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/c25db3f57c6037da05a2940246e76fd0cbd93c27/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/globalnav/GlobalNavigationPageObject.java#L122) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MINOR [Define a constant instead of duplicating this literal "Special:Search" 4 times.](https://github.com/Wikia/selenium-tests/blob/c25db3f57c6037da05a2940246e76fd0cbd93c27/src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/GlobalNavigationSearch.java#L19) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/dropdowncomponentobject/DropDownComponentObject.java [hunk 11]
```diff
@@ -79,9 +82,32 @@ public Boolean apply(WebDriver webDriver) {
     return this;
   }
 
+  public DropDownComponentObject openDropDownWithEntryPoint(final WebElement entryPoint) {
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/dropdowncomponentobject/DropDownComponentObject.java]: is `final` necessary?

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/dropdowncomponentobject/DropDownComponentObject.java [hunk 12]
```diff
@@ -79,9 +82,32 @@ public Boolean apply(WebDriver webDriver) {
...
+    driver.manage().timeouts().implicitlyWait(500, TimeUnit.MILLISECONDS);
+    try {
+      new WebDriverWait(driver, 20, 2000).until(new ExpectedCondition<Boolean>() {
+        @Override
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/dropdowncomponentobject/DropDownComponentObject.java]: is this necessary?

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/globalnav/GlobalNavigationPageObject.java [hunk 13]
```diff
@@ -165,9 +180,50 @@ private DropDownComponentObject getAccountNavigation() {
...
+  }
+
+  public DropDownComponentObject openExploreWikiaDropdown() {
+    return getExploreWikiaDropdownComponent().openDropDownWithEntryPoint(exploreWikiaDropdownEntryPoint);
+  }
+
+  public List<String> getDropdownLinks() {
+    List<String> linksLabels = new ArrayList<>();
+    List<WebElement> linksInDropdown = exploreWikiaDropdownComponent.getAllLinksInExploreWikiaDropdown();
+    for (WebElement link : linksInDropdown) {
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/globalnav/GlobalNavigationPageObject.java]: please provide extra line above loop

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/globalnav/GlobalNavigationPageObject.java [hunk 14]
```diff
@@ -165,9 +180,50 @@ private DropDownComponentObject getAccountNavigation() {
     return accountNavigation;
   }
 
+  private DropDownComponentObject getExploreWikiaDropdownComponent() {
+    if (exploreWikiaDropdownComponent == null) {
+      exploreWikiaDropdownComponent = new DropDownComponentObject(driver);
+    }
+    return exploreWikiaDropdownComponent;
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/globalnav/GlobalNavigationPageObject.java]: please provide extra line above `return`

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/globalnav/GlobalNavigationPageObject.java [hunk 15]
```diff
@@ -165,9 +180,50 @@ private DropDownComponentObject getAccountNavigation() {
...
+        linksLabels.add(link.getText());
+      }
+    }
+
+    return linksLabels;
+  }
+
+  public void closeDropdown() {
+    JavascriptExecutor js = (JavascriptExecutor) driver;
+    js.executeScript("arguments[0].classList.remove('active')", exploreWikiaDropdown);
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/globalnav/GlobalNavigationPageObject.java]: please use method directly, you don't need to store it in variable

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/globalnav/GlobalNavigationPageObject.java [hunk 16]
```diff
@@ -165,9 +180,50 @@ private DropDownComponentObject getAccountNavigation() {
...
+  }
+
+  public boolean areHubsLinksVisible() {
+    for (WebElement hubLink : hubLinks) {
+      if (!hubLink.isDisplayed()) {
+        return false;
+      }
+    }
+    return true;
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/globalnav/GlobalNavigationPageObject.java]: please provide extra line above

File: src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/ExploreWikiaDropdown.java (hunk 1 above)
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/ExploreWikiaDropdown.java]: please use `private final static` for class const

File: src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/ExploreWikiaDropdown.java (hunk 2 above)
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/ExploreWikiaDropdown.java]: please use `private final static` for class const

File: src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/ExploreWikiaDropdown.java (hunk 3 above)
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/ExploreWikiaDropdown.java]: please use `private final static` for class const

File: src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/ExploreWikiaDropdown.java (hunk 4 above)
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/ExploreWikiaDropdown.java]: please use `private static final` for class const

File: src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/ExploreWikiaDropdown.java [hunk 17]
```diff
@@ -0,0 +1,39 @@
...
+          Arrays.asList("Games", "Movies", "TV", "Top Communities", "Community Central", "START A WIKIA");
+
+  @Test(groups = {"TestExploreWikiaDropdown_001", "TestExploreWikiaDropdown", "GlobalNav"})
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/ExploreWikiaDropdown.java]: please move `TestExploreWikiaDropdown", "GlobalNav"` groups and put them on class

File: src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/ExploreWikiaDropdown.java [hunk 18]
```diff
@@ -0,0 +1,39 @@
...
+  public void TestExploreWikiaDropdown_001_dropdownIsPresent() {
+    HomePageObject homePage = new HomePageObject(driver);
+    GlobalNavigationPageObject globalNav = new GlobalNavigationPageObject(driver);
+    homePage.openWikiPage(this.wikiURL);
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/ExploreWikiaDropdown.java]: just `wikiUrl` should be enough

File: src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/GameStarLogo.java [hunk 19]
```diff
@@ -11,28 +11,16 @@
...
-    HomePageObject homePage = new HomePageObject(driver);
-    VenusGlobalNavPageObject globalNav = new VenusGlobalNavPageObject(driver);
-    homePage.openWikiPage(urlBuilder.getUrlForWiki(deWikiName));
-    homePage.resizeWindow(SHOW_LOGO_RESOLUTION);
-
-    Assertion.assertTrue(globalNav.isGameStarLogoDisplayed(), "GameStar Logo should be visible");
-  }
-
-  @Test(groups = {"TestGameStarLogo_002", "GameStarLogo", "GlobalNav"})
+  @Test(groups = {"TestGameStarLogo_001", "GameStarLogo", "GlobalNav"})
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/GameStarLogo.java]: please move groups `"GameStarLogo", "GlobalNav"` on class

File: src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/GameStarLogo.java [hunk 20]
```diff
@@ -11,28 +11,16 @@
...
   public void TestGameStarLogo_002_gameStarLogoNotPresentOnSmallResolution() {
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/GameStarLogo.java]: 002 --> 001

File: src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/GameStarLogo.java [hunk 21]
```diff
@@ -11,28 +11,16 @@
...
     HomePageObject homePage = new HomePageObject(driver);
-    VenusGlobalNavPageObject globalNav = new VenusGlobalNavPageObject(driver);
+    GlobalNavigationPageObject globalNav = new GlobalNavigationPageObject(driver);
     homePage.openWikiPage(urlBuilder.getUrlForWiki(deWikiName));
     homePage.resizeWindow(HIDE_LOGO_RESOLUTION);
 
     Assertion.assertFalse(globalNav.isGameStarLogoDisplayed(),
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/GameStarLogo.java]: you can cast method directly, don't need to store in var `globalNav`

File: src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/GlobalNavigationHubLinks.java [hunk 22]
```diff
@@ -2,18 +2,19 @@
...
-import com.wikia.webdriver.pageobjectsfactory.pageobject.globalnav.VenusGlobalNavPageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.globalnav.GlobalNavigationPageObject;
 
 import org.testng.annotations.Test;
 
 @Test(groups = {"HubLinksInGlobalNav"})
 public class GlobalNavigationHubLinks extends NewTestTemplate {
 
-  @Test(groups = {"TestHubLinksInGlobalNav_001", "GlobalNav"})
+  @Test(groups = {"TestHubLinksInGlobalNav_001", "GlobalNav"},
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/GlobalNavigationHubLinks.java]: move `"GlobalNav"` on class

File: src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/GlobalNavigationWikiaLogo.java [hunk 23]
```diff
@@ -24,10 +29,25 @@ public void TestWikiaLogoInGlobalNav_001_centralWikiExists(String wikiName,
...
     homePage.getUrl(urlBuilder.getUrlForWiki(wikiName));
-    homePage.getVenusGlobalNav().clickWikiaLogo();
+    homePage.getGlobalNavigation().clickWikiaLogo();
 
     PageObjectLogging.log("CHECK URL", "Expected: " + urlBuilder.getUrlForWiki(expectedCentralUrl),
         new WebDriverWait(driver, 10).until(ExpectedConditions.urlContains(urlBuilder
             .getUrlForWiki(expectedCentralUrl))));
   }
+
+  @Test(groups = {"TestFandomLogo_001", "GlobalNav"})
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/GlobalNavigationWikiaLogo.java]: move `"GlobalNav"` on class
001 --> 002

File: src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/GlobalNavigationWikiaLogo.java [hunk 24]
```diff
@@ -24,10 +29,25 @@ public void TestWikiaLogoInGlobalNav_001_centralWikiExists(String wikiName,
...
+  public void TestFandomLogo_001_logoPresentOnENCommunity() {
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/GlobalNavigationWikiaLogo.java]: 001 --> 002

File: src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/HubsLinks.java [hunk 25]
```diff
@@ -0,0 +1,28 @@
...
+
+  @Test(groups = {"TestHubsLinks_001", "GlobalNav"})
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/HubsLinks.java]: move `"GlobalNav"` on class

File: src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/HubsLinks.java [hunk 26]
```diff
@@ -0,0 +1,28 @@
...
+  public void TestHubsLinks_001_linksArePresent() {
+    HomePageObject homePage = new HomePageObject(driver);
+    GlobalNavigationPageObject globalNav = new GlobalNavigationPageObject(driver);
+    homePage.openWikiPage(this.wikiURL);
+
+    driver.manage().window().setSize(HUBS_OUTSIDE_DROPDOWN_RESOLUTION);
+
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/HubsLinks.java]: remove this line

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/globalnav/GlobalNavigationPageObject.java (hunk 14 above)
Comment (bognix) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/globalnav/GlobalNavigationPageObject.java]: yes, it's accessed inside inner class - it has to be final

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/dropdowncomponentobject/DropDownComponentObject.java (hunk 11 above)
Comment (bognix) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/dropdowncomponentobject/DropDownComponentObject.java]: yes, it's accessed inside inner class - it has to be final

File: src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/ExploreWikiaDropdown.java (hunk 18 above)
Comment (bognix) [inline src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/ExploreWikiaDropdown.java]: can I leave this? for me it's more obvious this way that this is a class property not some variable stored somewhere.

What is more I still see some usages of this.wikiURL

File: src/test/java/com/wikia/webdriver/testcases/globalnavigationbar/Loading.java [hunk 27]
```diff
@@ -0,0 +1,77 @@
...
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.SpecialWikiActivityPageObject;
+
+import org.openqa.selenium.Dimension;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Test(groups = {"globalnavigationbar", "Loading"})
+public class Loading extends NewTestTemplate{
```
Comment (bognix) [inline src/test/java/com/wikia/webdriver/testcases/globalnavigationbar/Loading.java]: "Loading" class name is very non-intuitive for me. Could you rename to something like "Interactions" or "presence" at least? Although I can't say those names describes what happens inside those classes as well

File: src/test/java/com/wikia/webdriver/testcases/globalnavigationbar/Loading.java [hunk 28]
```diff
@@ -0,0 +1,77 @@
...
+  private static final List<String> EXPECTED_LINKS_BIG_RESOLUTION =
+      Arrays.asList("Top Communities", "Community Central", "START A WIKIA");
+  private static final List<String> EXPECTED_LINKS_SMALL_RESOLUTION =
+      Arrays.asList("Games", "Movies", "TV", "Top Communities", "Community Central", "START A WIKIA");
+
+  private final static String deWikiName = "de.gta";
+  private static final Dimension HIDE_LOGO_RESOLUTION = new Dimension(1200, 720);
+
+  @Test(groups = {"globalnavigationbar_Loading_001"})
+  public void globalNavigationBarIsFixedOnScroll() {
```
Comment (bognix) [inline src/test/java/com/wikia/webdriver/testcases/globalnavigationbar/Loading.java]: why this naming convention? For me it was very convenient when the group was a part of method name.
This way when I was checking logs and I only see method names I could copy the first part of the name and run the test with this group - right now it's not possible without checking the code ;(

**Pull request merged**